bucket/simple-dnscrypt.json

```
@@ -0,0 +1,62 @@
+{
+    "version": "0.7.1",
+    "description": "An easy to use GUI for dnscrypt-proxy",
+    "homepage": "https://simplednscrypt.org/",
+    "license": "MIT",
+    "suggest": {
+        "vcredist2017": "extras/vcredist2017"
+    },
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt64.msi",
+            "hash": "5BA0C94774ACD7ADE18446B4351E7D28A948C490AEFC50DC6C08C13551F19F7F",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt64.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt64.msi\" | Out-Null}"
+            ]
+        },
+        "32bit": {
+            "url": "https://github.com/bitbeans/SimpleDnsCrypt/releases/download/0.7.1/SimpleDNSCrypt.msi",
+            "hash": "CAC6B8119AF3DA70A5977A2E73EE5214303990FEDD7ABBC5F247A1E2CF19A663",
+            "post_install": [
+                "if (Test-Path \"$dir\\SimpleDNSCrypt.msi\") {Remove-Item \"$dir\\SimpleDNSCrypt.msi\" | Out-Null}"
+            ]
+        }
+    },
+    "pre_install": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "",
+        "if (!(Test-Path \"$env:LOCALAPPDATA\\SimpleDnsCrypt\")) {New-Item -Path \"$env:LOCALAPPDATA\" -Name \"SimpleDnsCrypt\" -ItemType \"directory\" -Force | Out-Null}"
+    ],
+    "shortcuts": [
+        [
+            "SimpleDnsCrypt.exe",
+            "Simple DnsCrypt"
+        ]
+    ],
+    "bin": "SimpleDnsCrypt.exe",
+    "persist": [
+        "dnscrypt-proxy\\"
+    ],
+    "pre_uninstall": [
+        "if (!(is_admin)) { error \"$app requires admin rights to $cmd\"; break }",
+        "",
+        "if (Get-Process -Name \"SimpleDnsCrypt\") {Stop-Process -Name \"SimpleDnsCrypt\" | Out-Null}",
+        "if (Get-Service -Name \"dnscrypt-proxy\") {Stop-Service -Name \"dnscrypt-proxy\"| Out-Null; Remove-Service -Name \"dnscrypt-proxy\" | Out-Null}",
+        "if (Test-Path \"$dir\") {Remove-Item \"$dir\" -Force -Recurse | Out-Null}",
+        "if (Test-Path \"$persist_dir\") {Remove-Item \"$persist_dir\" -Force -Recurse | Out-Null}"
```

```suggestion
```

I don't think this is needed. These folders should be automatically removed or persisted.

---

> ```suggestion
> 
> ```
> 
> 
> 
> I don't think this is needed. These folders should be automatically removed or persisted.

When I tried uninstalling the app without that part of the script, I sometimes get an error that says `ERROR couldn't remove '~\scoop\apps\simple-dnscrypt\current' it may be in use`